### ISSUE TITLE
bumblebee init script fails with: unrecognized option '--pidfile'

### ISSUE THREAD

**Issue by arisudesu** [manjaro/packages-openrc#4]
As title says, bumblebeed cannot be started and produces this error, because it is built with ```--without-pidfile``` in Arch and Manjaro.

**Comment by udeved** [comment]
Thx for reporting.
I had suspected something like that.
It means, we would need a separate bumblebee package.

**Comment by arisudesu** [comment]
Why the separate package? Why not just patch bumblebee init script?

**Comment by udeved** [comment]
Because initscript needs pid ability from bumblebee, and --make-pid didn't work for start-stop-daemon because bublebee runs without pid.
Hence you now have a package which start bumblebee with pid.

**Comment by arisudesu** [comment]
This is very strange. I managed to run bumblebeed with no pidfile with this line:
```start-stop-daemon -S -b -m -p "${PIDFILE}" -x /usr/sbin/bumblebeed -- --use-syslog ${BUMBLEBEE_EXTRA_OPTS}```
I replaced the ```-D``` of bumblebeed with explicit ```---use-syslog``` to run it in foreground but keep logging to syslog, and added ```-b``` and ```-m``` to start-stop-daemon to run it in background and make pidfile on its own. All seems to work fine with this. I see no reason to maintain a separate package, because it works with the native package.
